Done!

mshecket opened the pull request:
I thought this one was a lot easier than the last one!

mshecket committed:
Done!

sjreich reviewed (commented):
Do the tests actually pass here?  Seems like this isn't really getting what's intended by the `canonical` method`

mshecket commented:
That's very possible! The word kind of flew over my head, so I probably need to go back and learn what that's about. Thanks!

mshecket commented:
No wait--I know what is it...I confused anagram and palindrome. DUUUUHHHH

I'll get back to this soon and do it for real.

mshecket committed:
Now actually checking for anagrams

I was mistakenly checking for palindromes instead of anagrams, but now I've got it working correctly.
